src/containers/TrashToolbar.jsx

```diff
@@ -10,13 +10,11 @@ import Menu, { MenuButton, Item } from 'react-bosonic/lib/Menu'
 import { showSelectionBar } from '../actions'
 import { mustShowSelectionBar } from '../reducers'
 
-const TrashToolbar = ({ t, error, disableFolderCreation, isSelectionBarVisible, showSelectionBar }) => (
+const TrashToolbar = ({ t, error, isSelectionBarVisible, showSelectionBar }) => (
   <div className={styles['fil-toolbar']} role='toolbar'>
-    <div className={styles['fil-toolbar-trash']}>
-      <button className={classNames(styles['danger-outline'], styles['coz-btn--delete'])}>
-        {t('toolbar.delete_all')}
-      </button>
-    </div>
+    <button className={classNames(styles['fil-toolbar-trash'], 'coz-btn', 'coz-btn--danger-outline')}>
+      {t('toolbar.delete_all')}
+    </button>
```

Comment:
I noticed this and the associated css changes, but the button doesn't have the correct appearance after these changes.